test_scripts/Defects/6_1/3321_OnAppPropertiesChange_remove_nickname.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3321
+--
+-- Description:
+-- Processing of the OnAppPropertiesChange notification to HMI on SetAppProperties request without one of nicknames
```

Comment:
@VjKlepikov I would propose to rephrase description to it reflects better the goal of script:
```
Processing of the OnAppPropertiesChange notification to HMI on SetAppProperties request without one of nicknames
```
=>
```
Processing of the OnAppPropertiesChange notification to HMI on SetAppProperties request with updated data of application where one of nicknames had been removed
```